JRIngram/maze-gen
# Feature/hunt and kill

# Pull Request (PR) - Hunt And Kill
## Description of Change
Add the hunt and kill algorithm as an option to be used to generate mazes


## Link To Issue 



## How to test this change
Run the unit tests
Run the linter
Add npm package to a project and generate a maze with both DEPTHFIRST and HUNTANDKILL and check that mazes are valid


## Checklists
### Checklist for developer of feature

### Checklist for reviewers
